src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --hover-link-color: #00acdc;
   margin: 0;
 }
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.header-nav {
+  align-items: center;
+  display: flex;
+}
+
+.nav-list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav-list-item {
+  display: flex;
+  align-items: center;
+}
+
+.nav-link {
+  display: inline-block;
+  text-align: center;
+  text-wrap: nowrap;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 60px;
```

Comment:
Don't just copy all styles from Figma. Think, which of them are relevant. Uneven sizes (e.g. line-height: 14.6px) are definitely useless.